test/topology/conftest.py

```diff
@@ -54,10 +54,16 @@ def _wrap_future(f: ResponseFuture) -> asyncio.Future:
     aio_future = loop.create_future()
 
     def on_result(result):
-        loop.call_soon_threadsafe(aio_future.set_result, result)
+        if not aio_future.done():
```

Comment:
Wut

how did this manifest? What happened? Was there an exception or something?

Reply:
When there are timeouts for some reason the driver's future calls our callback `on_result()` with a `None` param. It pollutes the logs when trying to figure out what is happening, so I just handle it here.